hack/collapse/auto_collapse.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# Auto Collapse
+#This is a script that takes in
+# - a markdown file
+# - a designated output file
+# - the number of lines as a threshold on when to collapse a section.
+#
+# Sample
+# ./auto_collapse.sh test_example.md test_example_collapsible.md 10
```

Comment:
```suggestion
# ./auto_collapse.sh test_example.md test_example_collapsible.md 5
```
That's absolute nitpicking, but just to avoid confusion, since the example file does not have a list that's 10 entries long and therefore, the script has no effect.